### PR TITLE
Added Path normalizing

### PR DESCRIPTION
**Pull request by LokiMidgard**
I tried to add some path normalizing but at least on my machine it does not recognize `path`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "chokidar": "3.4.0",
     "dedent-js": "1.0.1",
     "glob": "7.1.6",
-    "loader-utils": "2.0.0"
+    "loader-utils": "2.0.0",
+    "path": "0.12.7"
```
Comment (macabeus) [inline package.json:26]: I think that isn't  necessary to add this dependency, because it's built-in on Node and we are already using Node.

Comment (macabeus) [reply package.json:26]: Removing that, I didn't have the "error that path.normalize is undefined"

**Comment by macabeus** [comment]
I checked your PR and your approach, but I think that, instead of fixing it on `updateGlobalState` function, would be better to fix it on `cli/index.js`, in order to not send paths mixing Unix and Windows' delimiters.

Probably will be necessary to update these lines:
```js
  glob('**/*.ftl', { ignore: ['node_modules/**/*', '.git/**/*'] }, (errors, matches) => {
    const files = matches.map(path => ({ // this path
```

and
```js
  watcher
    .on('ready', () => console.log('🎬 Ready!'))
    .on('change', (path) => {
      console.log(`🔍 File was changed: ${path}`) // this path
```

Notice that firstly we get the path using [`glob`](https://www.npmjs.com/package/glob) and then, to watch the files, we are using [`chokidar`](https://github.com/paulmillr/chokidar). So we'll need to normalize the paths returned to use the same delimiters here.

Probably there is a solution out-of-the-box on these libraries.

**Comment by LokiMidgard** [comment]
`chokidar` normalizes the path already, but I havn't found an option for `glob`. I normalize the path now in the `map` function

**Comment by macabeus** [comment]
Thanks.  I tested on Windows and macOS, and looks like that everything is working fine.
I only updated your commit to import with `const { normalize } = require('path')` instead of `const nodePath = require('path')`, as well as the commit author.

**Pull request merged**